### PR TITLE
Update device-enrollment-shared-ipad.md

### PR DESCRIPTION
**Pull request by msbemba**
In the known limitations section added info that the  available apps are not supported

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2842

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/device-enrollment-shared-ipad.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ The following are known limitations when working with shared iPads:
...
 - **App Store installations are disabled:** The App Store is available by default on Shared iPad. But app installation is disabled for App Store apps when a device is set up as a Shared iPad. It is recommended that you disable App Store using configuration profiles in Intune.
-- **Company Portal and available apps are not supported:** Intune Company Portal app, the Intune Company Portal website are not supported on Shared iPads. Apps must be assigned as “required” to device groups containing the Shared iPad to install. Available apps are supported on Shared iPad.
+- **Company Portal and available apps are not supported:** Intune Company Portal app, the Intune Company Portal website are not supported on Shared iPads. Apps must be assigned as “required” to device groups containing the Shared iPad to install. Available apps are not supported on Shared iPad.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/device-enrollment-shared-ipad.md:131]:
```suggestion
- **Company Portal and available apps are not supported:** Intune Company Portal app and the Intune Company Portal website are not supported on Shared iPad. Apps must be assigned as “required” to device groups containing the Shared iPad to install. Available apps are not supported on Shared iPad.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**